src/componentTest/resources/expectedTemplateContent/performance-tests/performance-tests/src/main/scala/com/blackbaud/service/ServiceTest.scala

```diff
@@ -0,0 +1,84 @@
+package com.blackbaud.service
+
+import com.blackbaud.gatling.bbauth.BBAuthHelper
+import io.gatling.core.Predef.{exec, forever, scenario}
+import io.gatling.http.Predef.http
+import io.gatling.core.Predef._
+import io.gatling.core.structure.{ChainBuilder, ScenarioBuilder}
+import io.gatling.http.protocol.HttpProtocolBuilder
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class ServiceTest extends Simulation {
+
+  var maxTestRuntimeSec: Int = Option(System.getProperty("max.test.runtime.sec")).map(_.toInt).getOrElse(30)
```

Comment:
Why not use com.blackbaud.gatling.SimulationConfiguration for these properties?

Reply:
Thanks @blackbaud-alanacable. Just forgot it was a thing.